TROUBLESHOOTING.md

```diff
@@ -0,0 +1,67 @@
+If you have a non-library SendGrid issue, please contact our [support team](https://support.sendgrid.com).
+
+If you can't find a solution below, please open an [issue](https://github.com/sendgrid/open-source-library-data-collector/issues).
+
+## Table of Contents
+
+* [Environment Variables and Your SendGrid API Key](#environment)
+* [Error Messages](#error)
+* [Version Convention](#versions)
+
+<a name="environment"></a>
+## Environment Variables and Your SendGrid API Key
+
+### Initial Setup
+
+* copy the .env_sample file to .env
+```bash
+cp .env_sample .env
+```
+
+* open and add the params into this file (.env)
+
+* then source the file
+```bash
+source ./.env
+```
+
+All of our examples assume you are using [environment variables](https://github.com/sendgrid/sendgrid-python#setup-environment-variables) to hold your SendGrid API key.
```

Comment:
Can you please update the repo path here?

https://github.com/sendgrid/open-source-library-data-collector#environment-variables